pkg/backup/backup.go

```diff
@@ -116,6 +116,11 @@ func New(kclient kubernetes.Interface, clusterName, ns string, sp spec.ClusterSp
 	}, nil
 }
 
+// MakePrefix concatenates s3path, clusterName, and namespace.
+func MakePrefix(s3path, clusterName, namespace string) string {
+	return path.Join(s3path, clusterName, namespace)
```

Comment:
v1?

Reply:
isn't the point of supporting `s3path` to allow customize s3 path like `bucket_name\s3path\...` ?

Reply:
yes.
But why do you remove v1?

Reply:
maybe I am missing something.

Should `v1` be somewhere the the `bucket_name\s3path\...` like `bucket_name\v1\s3path\...`  or `bucket_name\"v1+s3path"\...`